integrations/sharepoint-online/schema.zod.ts

```diff
@@ -79,3 +79,9 @@ export const fetchFileSchema = z.object({
     id: z.string(),
     download_url: z.string().nullable()
 });
+
+export const folderContentInputSchema = z.object({
```

Comment:
The should be automatically generated using 

npm run generate:zod --integration=sharepoint-online